test/e2e/tests/contract-interactions.spec.js

```diff
@@ -11,64 +11,40 @@ describe('Deploy contract and call contract methods', function () {
       },
     ],
   };
+  const smartContract = 'piggybank';
```

Comment:
Can you use [this](https://github.com/MetaMask/metamask-extension/blob/develop/test/e2e/seeder/smart-contracts.js#L39) instead of the hardcoded string?

Reply:
SMART_CONTRACTS.PIGGYBANK

Reply:
sure, adding this. Ty!!